### DISCUSSION TITLE
Tabufline , closeAllbufs() function is giving an error 🥲

### DISCUSSION THREAD

**Discussion by mnkhod** [NvChad/NvChad#1964] [Q&A] [answered]
Anybody else having this problem? 
![image](https://user-images.githubusercontent.com/46494779/234787835-662ac27e-4061-45bf-b557-e6bd2a16a483.png)

Here is the mapping in my config
![image](https://user-images.githubusercontent.com/46494779/234787920-5a4a76a9-e9a2-4610-afdd-f4e859be3ab9.png)

**Comment by siduck** [comment, marked as answer]
hi @mnkhod sorry there's a typo in docs. its Bufs and not bufs

**Reply by mnkhod** [reply]
ohhhhh

**Reply by mnkhod** [reply]
Can i contribute on that by making a pull request? idk where the front-end repo is at but do wanna help 🫡

**Reply by siduck** [reply]
hi @mnkhod, really appreciate the help! But I had already corrected the docs https://github.com/NvChad/nvchad.github.io/commit/8af43a276dd557bef83b3550309df86cb264f7ff
